frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tsx

```diff
@@ -763,6 +763,70 @@ const TAGGED_TEMPLATES = {
       </>
     );
   },
+  createDeclarationProfile: (activity: IActivity, isPremiumTier: boolean) => {
+    let teamText = "";
+    if (!isPremiumTier) {
+      teamText = "all macOS hosts";
+    }
```

Comment:
Should this be switch or `else if` chained with conditional `teamText`below?

Reply:
Also it seems like this can be extracted to a helper function and used for the other three DDM activity items?